ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -1306,13 +1306,15 @@ class JITModBuilder {
               continue;
             }
 
-            // Vararg related intrinsics appear to be always inlined.
-            if (IID == Intrinsic::vastart || IID == Intrinsic::vaend)
-              continue;
-
-            // Other intrinsics that appear to be always inlined.
-            if (IID == Intrinsic::smax || IID == Intrinsic::usub_with_overflow)
+            // Whitelist intrinsics that appear to be always inlined.
```

Comment:
If the set of inlined intrinsics grows (it surely will), we should make a constant collection at the top of the file that we can check for membership.

Reply:
Want me to add it here?

Reply:
I leave that up to you. Arguably not using a collection performs better.

Reply:
Let's keep it for now until we have more function and then we can decide if there's a better way.